### PR TITLE
File collector

### PR DESCRIPTION
**Pull request by stepanstipl**
This PR adds support for local file collector #6 via the `-f` flag. Supports JSON and YAML manifests, flag can be repeated multiple times:
```
$./kubent -f manifest-1.yaml -f manifest-2.json
```

_Support for reading from stdin will be in a separate PR._

Should be merged after #14 .

### CODE REVIEW

**Comment by david-doit-intl** [comment]
I like this, adding a file provider is good.

**Comment by david-doit-intl** [comment]
LGTM, I would say about the tests should be added

**Comment by david-doit-intl** [comment]
make sure you rebase!

**Comment by stepanstipl** [comment]
@david-doit-intl added tests, pls. have a look 👍

File: fixtures/meow.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+🐈
```
Comment (david-doit-intl) [inline fixtures/meow.txt:1]: :+1:

**Pull request merged**